package.json

```diff
@@ -69,7 +69,7 @@
 		"electron": "29.1.0",
 		"electron-devtools-installer": "^3.2.0",
 		"electron-playwright-helpers": "^1.7.0",
-		"eslint": "^8.0.1",
+		"eslint": "^8.57.1",
```

Comment:
Not something we need to address in this PR, but noting that https://eslint.org/ says ESLint 8.x is EOL, so we should look into [updating to 9.x](https://eslint.org/docs/latest/use/migrate-to-9.0.0).